src/realm/query_expression.hpp

```diff
@@ -2279,6 +2410,11 @@ class Columns<Link> : public Subexpr2<Link> {
         return LinkCount(m_link_map);
     }
 
+    LinkCount size() const
+    {
+        return LinkCount(m_link_map);
```

Comment:
I think using "count" for the size of a string would be misleading. The reason for keeping "count" in a LinkList column is to have backward compatibility. Having two methods returning the same thing is not such a big deal in this case - I think. We also have both "length" and "size" on std::string.